### DISCUSSION TITLE
[English]: Security as a Service (SecaaS)

### DISCUSSION THREAD

**Discussion by turjachaudhuri** [cncf/glossary#408] [General]
### Details (optional)

This is a very important concept in the modern cloud ecosystem .

Security as a Service is missing in the glossary , and will be a good addition .

**Comment by turjachaudhuri** [comment]
If the issue is accepted will like to work on this .

**Comment by JasonMorgan** [comment]
@turjachaudhuri do you have some examples of this term being used? I haven't seen it before.

**Comment by turjachaudhuri** [comment]
> @turjachaudhuri do you have some examples of this term being used? I haven't seen it before.

I have seen it used in some discussions with clients , also found it in blogs -

https://en.wikipedia.org/wiki/Security_as_a_service#:~:text=Security%20as%20a%20service%20(SECaaS,cost%20of%20ownership%20is%20considered.
https://cloudsecurityalliance.org/research/working-groups/security-as-a-service/

**Comment by JasonMorgan** [comment]
Hey folks! Thanks for reading this far. As of right now SecaaS is a term that we aren't sure is general purpose and may represent a new marketing instead of technical term. Please let us know your thoughts.

Thanks!

**Comment by vidipsingh** [comment]
Hi, I am new to open source,
Can someone guide me on how things are done in Open Source?
